app/validators/capital_assessment/property_ownership_validator.rb

```diff
@@ -0,0 +1,54 @@
+module CapitalAssessment
+  class PropertyOwnershipValidator < ActiveModel::Validator
+    attr_reader :form
+
+    def validate(form)
+      @form = form
+
+      form.errors.add :percentage_applicant_owned, :invalid unless valid_ownership_total?(form)
+      form.errors.add :percentage_partner_owned, :invalid if client_has_partner(form) && !valid_ownership_total?(form)
+    end
+
+    private
+
+    def valid_ownership_total?(form)
+      return true if yet_to_add_other_owners?(form)
+      return true unless complete_property_owners?(form)
```

Comment:
We want to be able to go to the next screen if property owners are incomplete (via the save and come back later button) so the user can go back to the other property owners screen and enter the ownership information